send-email/send-email.py

```diff
@@ -0,0 +1,49 @@
+import time
+from azure.communication.email import EmailClient, EmailContent, EmailAddress, EmailMessage, EmailRecipients
+
+def main():
+    try:
+        connection_string = "<ACS_CONNECTION_STRING>"
+        client = EmailClient.from_connection_string(connection_string)
+        sender = "<SENDER_EMAIL>"
+        content = EmailContent(
+            subject="Test email from Python",
+            plain_text="This is plaintext body of test email.",
+            html= "<html><h1>This is the html body of test email.</h1></html>",
+        )
```

Comment:
@yogeshmo, how does this compare with the model-less SDK?

Reply:
The model-less SDK will be quite different. These objects will be defined as dictionaries instead of objects. For now the model-less one isn't public, so I can go back and change these when it is. It shouldn't take too long.